src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -3297,6 +3297,10 @@ private unsafe void ParseRemaining()
                     {
                         _string = _syntax.SchemeName + SchemeDelimiter;
                     }
+
+                    _info.Offset.Scheme = 0;
+                    _info.Offset.User = (ushort)_string.Length;
```

Comment:
Is it possible to craete a test that exercises the case that needs the `User` offset updated?